ecosystem/platform/server/app/models/user.rb

```diff
@@ -116,8 +117,13 @@ def email_required?
     false
   end
 
-  # This is to allow username instead of email login in devise (for aptos admins)
-  def will_save_change_to_email?
-    false
+  # Use mailchimp instead of the default devise confirmation email.
+  def send_confirmation_instructions
+    generate_confirmation_token! unless @raw_confirmation_token
+
+    url_options = Rails.application.config.action_mailer.default_url_options
+    url = Rails.application.routes.url_helpers.user_confirmation_url(**url_options,
+                                                                     confirmation_token: @raw_confirmation_token)
```

Comment:
is `raw_confirmation_token` something Devise adds?

Reply:
Yes, the first half of the method is copied from the original implementation in Devise.